src/libs/utils.js

```diff
@@ -240,3 +240,10 @@ export const convertValue = _.curry((type, value) => {
 export const normalizeLabel = _.flow(_.camelCase, _.startCase)
 
 export const kvArrayToObject = _.reduce((acc, { key, value }) => _.set(key, value, acc), {})
+
+export const isValidWsExportTarget = _.curry((sourceWs, destWs) => {
```

Comment:
This looks correct to me. I'd love to see some test coverage of this because it's pretty important to get right.

More important... is this the only guard? If so, we need to fix something server-side to make sure this can't be done.